build.zig

```diff
@@ -103,16 +105,17 @@ pub fn build(b: *std.Build) void {
         };
 
         for (examples) |name| {
+            const mod = b.createModule(.{
+                .target = target,
+                .optimize = optimize,
+            });
             const exe = b.addExecutable(.{
                 .name = name,
-                .root_module = b.createModule(.{
-                    .target = target,
-                    .optimize = optimize,
-                }),
+                .root_module = mod,
```

Comment:
This makes the diff unnecessarily complicated. Can you keep the `b.createModule` call inside the `b.addExecutable` call and then just replace `exe.addSomething` with `exe.root_module.addSomething`.